js/components/controlBar.js

```diff
@@ -327,91 +330,110 @@ var ControlBar = React.createClass({
         tabIndex="0"
         aria-label={playPauseAriaLabel}
         autoFocus={this.props.controller.state.playPauseButtonFocused}>
-        <Icon {...this.props} icon={playIcon} style={dynamicStyles.iconCharacter} />
-      </button>,
-
-      "live": <a className={liveClass}
+        <Tooltip enabled={isTooltipEnabled}
+                 alignment="left"
+                 responsivenessMultiplier={this.responsiveUIMultiple} bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height}
+                 text={this.props.playerState == CONSTANTS.STATE.PLAYING ? this.props.localizableStrings[this.props.language].Pause :
```

Comment:
Is there a reason you are using localizableStrings rather than Utils.getLocalizedString?